source/opt/ir_loader.cpp

```diff
@@ -23,23 +23,72 @@
 #include "source/opt/reflect.h"
 #include "source/util/make_unique.h"
 
+static const uint32_t kExtensionInstructionIndex = 4;
```

Comment:
These should try to match the operand names in the spec.  A better name might be `kExtInstSetIndex`. 
 See https://www.khronos.org/registry/spir-v/specs/1.2/SPIRV.html#OpExtInst